src/Symfony/Component/Webhook/Server/JsonBodyConfigurator.php

```diff
@@ -20,14 +20,22 @@
  */
 final class JsonBodyConfigurator implements RequestConfiguratorInterface
 {
-    public function __construct(
-        private readonly SerializerInterface $serializer,
-    ) {
+    private PayloadEncoderInterface $payloadEncoder;
+
+    public function __construct(SerializerInterface|PayloadEncoderInterface|null $payloadEncoder = null)
```

Comment:
should we deprecate passing an instance of `SerializerInterface` as well? I think the final signature in 8.0 could be just `PayloadEncoderInterface $payloadEncoder`

Reply:
see https://github.com/symfony/symfony/pull/57881#discussion_r1696559863